temporian/core/evaluator.py

```diff
@@ -73,10 +100,25 @@ def evaluate(
         )
         for input_event, input_event_spec in input_data.items()
     }
-    # evaluate schedule
+
+    # Evaluate schedule
     outputs = evaluate_schedule_fn(materialized_input_data, schedule)
 
-    return {event: outputs[event] for event in normalized_query}
+    # Convert the result "outputs" into the same format as the query.
+    if query_type == Q_TYPE_DICT:
```

Comment:
The definition of the `Q_TYPE` enum/constants seems a bit overkill to me if the only purpose is to be able to do this comparison instead of doing using `isinstance(query, X)` again